core/src/script/CGXP/plugins/Editing.js

```diff
@@ -403,7 +412,24 @@ cgxp.plugins.Editing = Ext.extend(gxp.plugins.Tool, {
     /** private: method[addEditingLayer]
      */
     addEditingLayer: function() {
+        var defaultSelectStyle = OpenLayers.Feature.Vector.style['select'];
+        var self = this;
+        var context = {
+            getColor: function(feature) {
+                return self.selectionColors &&
+                    self.selectionColors[feature.attributes.__layer_id__] ||
+                    defaultSelectStyle.strokeColor;
+            }
+        }
+        var template = {
+            fillColor: "${getColor}",
+            strokeColor: "${getColor}"
+        }
+        OpenLayers.Util.applyDefaults(template, defaultSelectStyle);
+        var style = new OpenLayers.Style(template, {context: context});
         var editingStyleMap = new OpenLayers.StyleMap({
+            'default': style,
+            'select': style,
```

Comment:
This is what I think is better.
Actually in the editing interface, you usually (almost) never see the "default" style since the features are always selected or currently drawn (which corresponds to a specific style that we may want to keep.